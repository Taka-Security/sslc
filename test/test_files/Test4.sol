contract A {
  struct StructA {
    uint8 var1;
    uint var2;
    address var3;
  }
  struct StructB {
    bool var1;
    uint var2;
    bool var3;
    uint var4;
    bool var5;
    bytes32 var6;
    bool var7;
  }
}
contract B {
  struct StructA {
    uint8 var1;
    bytes32 var2;
    uint24 var3;
    bytes17 var4;
  }
}