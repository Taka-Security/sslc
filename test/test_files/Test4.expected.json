[
    {
        "file": "Test4.sol",
        "contract": "A",
        "struct": "StructA",
        "slotsSaved": 1,
        "timedout": false
    },
    {
        "file": "Test4.sol",
        "contract": "A",
        "struct": "StructB",
        "slotsSaved": 3,
        "timedout": false
    },
    {
        "file": "Test4.sol",
        "contract": "B",
        "struct": "StructA",
        "slotsSaved": 1,
        "timedout": false
    }
]